### PR TITLE
Make sure tests use timezones in time specific tests

### PR DESCRIPTION
**Pull request by cwrw**
Allow tests to utilise timezones, as running tests from a different timezone is causing issues if these aren't set.

### CODE REVIEW

**Review by martyn-w — APPROVED** [review]
👍 LGTM

File: spec/models/activity_spec.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
     let(:expected_attributes) do
       {
         id: nomis_scheduled_event['eventId'],
-        start_time: Time.parse(nomis_scheduled_event['startTime']),
+        start_time: Time.parse(nomis_scheduled_event['startTime']), # rubocop:disable Rails/TimeZone
```
Comment (cwrw) [inline spec/models/activity_spec.rb:35]: From the comment on the activity class:
Nomis startTime does not include a zone so we shouldn't use our local zone with Time.zone.parse and arbitraily add a timezone

File: spec/models/nomis_court_hearing_spec.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@
     let(:expected_attributes) do
       {
         id: nomis_court_hearing['id'],
-        start_time: Time.parse(nomis_court_hearing['dateTime']),
+        start_time: Time.zone.parse(nomis_court_hearing['dateTime']),
```
Comment (smoothcontract) [inline spec/models/nomis_court_hearing_spec.rb:24]: As this is also coming from Nomis, does the same caveat apply as above with `activity_spec` ?

**Review by smoothcontract — COMMENTED** [review]
Just one query on another spec related to Nomis but otherwise looks good 👍

File: spec/models/nomis_court_hearing_spec.rb (hunk 2 above)
Comment (cwrw) [reply spec/models/nomis_court_hearing_spec.rb:24]: good question, it looks like this only applies to activity, as in the court hearing we are explicitly passing in the zone:
https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/master/app/models/nomis_court_hearing.rb#L11

Comment (smoothcontract) [reply spec/models/nomis_court_hearing_spec.rb:24]: Cool, thanks for checking 👍

**Review by smoothcontract — APPROVED** [review]
LGTM 👍

**Pull request merged**